lightningd/peer_control.c

```diff
@@ -452,7 +452,7 @@ void add_peer(struct lightningd *ld, u64 unique_id,
 	peer->remote_funding_locked = false;
 	peer->scid = NULL;
 	peer->seed = NULL;
-	peer->balance = NULL;
+	peer->our_msatoshi = NULL;
```

Comment:
Because it's undefined until we have a channel open; we check this before dumping in getpeers.